tests/app/views/contexts/test_submit_context.py

```diff
@@ -0,0 +1,165 @@
+from unittest.mock import Mock, patch
+
+from app.data_models.answer_store import AnswerStore
+from app.data_models.list_store import ListStore
+from app.data_models.progress_store import ProgressStore
+from app.utilities.schema import load_schema_from_name
+from app.views.contexts import SubmitQuestionnaireContext
+from tests.app.app_context_test_case import AppContextTestCase
+
+
+class TestStandardSummaryContext(AppContextTestCase):
+    def setUp(self):
+        super().setUp()
+        self.metadata = {
+            "return_by": "2016-10-10",
+            "ref_p_start_date": "2016-10-10",
+            "ref_p_end_date": "2016-10-10",
+            "ru_ref": "def123",
+            "response_id": "abc123",
+            "ru_name": "Mr Cloggs",
+            "trad_as": "Samsung",
+            "tx_id": "12345678-1234-5678-1234-567812345678",
+            "period_str": "201610",
+            "employment_date": "2016-10-10",
+            "collection_exercise_sid": "789",
+            "schema_name": "0000_1",
+        }
+        self.language = "en"
+        self.answer_store = AnswerStore()
+        self.list_store = ListStore()
+        self.progress_store = ProgressStore()
+
+    def check_context(self, context):
+        self.assertEqual(len(context), 1)
+        self.assertTrue("summary" in context, "Key value summary missing from context")
+
+        summary_context = context["summary"]
+        for key_value in ("groups", "answers_are_editable", "summary_type"):
+            self.assertTrue(
+                key_value in summary_context,
+                f"Key value {key_value} missing from context['summary']",
+            )
+
+    def check_summary_rendering_context(self, summary_rendering_context):
+        for group in summary_rendering_context["summary"]["groups"]:
+            self.assertTrue("id" in group)
+            self.assertTrue("blocks" in group)
+            for block in group["blocks"]:
+                self.assertTrue("question" in block)
+                self.assertTrue("title" in block["question"])
+                self.assertTrue("answers" in block["question"])
+                for answer in block["question"]["answers"]:
+                    self.assertTrue("id" in answer)
+                    self.assertTrue("value" in answer)
+                    self.assertTrue("type" in answer)
+
+    @patch("app.jinja_filters.flask_babel.get_locale", Mock(return_value="en_GB"))
+    def test_submit_context_with_default_submission_content(self):
+        schemas = ["test_summary", "test_instructions"]
```

Comment:
I don't think we need to test this with two different schemas.

Reply:
It was to test when a summary was used and when a summary wasn't but as you pointed out, since that is done regardless of whether a summary exists or not, I have removed one redundant schema.